src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.html

```diff
@@ -18,7 +18,7 @@ <h2>
         {{ 'artemisApp.exam.studentReviewEnabled' | artemisTranslate }}
     </a>
 </div>
-<div class="mb-4" *ngIf="studentExam && studentExam.exercises && studentExam.exam && studentExam.exam.course">
+<div class="mb-4 container" style="max-width: 980px" *ngIf="studentExam && studentExam.exercises && studentExam.exam && studentExam.exam.course">
```

Comment:
... we should avoid inline styles. :)

Reply:
Mhm, yes that would be a good idea :) 883ee58ad9ce2b357b9acb3fd804d327f09e04b4